allensdk/brain_observatory/ecephys/ecephys_api/ecephys_nwb_api.py

```diff
@@ -68,7 +71,9 @@ def get_units(self) -> pd.DataFrame:
         return units_table
 
     def __get_full_units_table(self) -> pd.DataFrame:
```

Comment:
why double underscore?

Reply:
single underscore makes more sense I think. Thanks for the catch.